book/06-github/sections/2-contributing.asc

```diff
@@ -260,160 +260,161 @@ Isa sa mga magagandang bagay tungkol sa Git ay maaari mong patuloy na gawin iyon
+Sa wakas, maaari ka ring gumamit ng emoji sa iyong mga komento. Talagang ginagamit ito nang lubos sa mga komento na nakikita mo sa maraming mga isyu ng GitHub at mga Kahilingan na Pull. Mayroon ding isang emoji helper sa GitHub. Kung nagta-type ka ng komento at nagsimula ka ng isang `:` na karakter, tutulungan ka ng autocompleter na makita kung ano ang iyong hinahanap.
 
 [[_md_emoji_auto]]
 .Emoji autocompleter in action.
 image::images/markdown-06-emoji-complete.png[Emoji autocompleter]
 
-Emojis take the form of `:<name>:` anywhere in the comment. For instance, you could write something like this:
+Ang Emojis ay kumukuha sa form ng `:<name>:` saanman sa komento. Halimbawa, maaari kang magsulat ng ilang bagay tulad nito:
 
 [source,text]
 ----
-I :eyes: that :bug: and I :cold_sweat:.
+Ako ay :eyes: sa :bug: at ako'y :cold_sweat:.
 
-:trophy: for :microscope: it.
+:trophy: para :microscope: ito.
 
-:+1: and :sparkles: on this :ship:, it's :fire::poop:!
+:+1: at :sparkles: sa :ship: na ito, ito'y :fire::poop:!
 
 :clap::tada::panda_face:
 ----
 
-When rendered, it would look something like <<_md_emoji>>.
+Kapag nagawa, ito ay magiging tulad ng <<_md_emoji>>.
 
 [[_md_emoji]]
-.Heavy emoji commenting.
+.Mabibigay na pagkokomento na emoji.
 image::images/markdown-07-emoji.png[Emoji]
 
-Not that this is incredibly useful, but it does add an element of fun and emotion to a medium that is otherwise hard to convey emotion in.
 
+Hindi na ito ay hindi kapani-paniwala kapaki-pakinabang, ngunit ito ay nagdaragdag ng isang elemento ng kasiyahan at damdamin sa isang daluyan na kung hindi man ay mahirap upang ihatid ang damdamin.
 
-[NOTE]
+
+[TANDAAN]
 ====
-There are actually quite a number of web services that make use of emoji characters these days. A great cheat sheet to reference to find emoji that expresses what you want to say can be found at:
+Mayroon talagang maraming bilang ng mga serbisyo ng web na gumagamit sa mga araw na ito ng mga karakter na emoji. Isang magaling na sheet ng panlilinlang na magamit upang mahanap ang emoji na nagpapahayag kung ano ang gusto mong sabihin ay matatagpuan sa:
 
 http://www.emoji-cheat-sheet.com
 ====
 
-===== Images
+===== Mga Larawan
 
-This isn't technically GitHub Flavored Markdown, but it is incredibly useful. In addition to adding Markdown image links to comments, which can be difficult to find and embed URLs for, GitHub allows you to drag and drop images into text areas to embed them.
+Hindi ito Pinalasang Markdown ng GitHub, ngunit ito ay hindi kapani-paniwala kapaki-pakinabang. Bilang karagdagan sa pagdaragdag ng mga link ng larawan ng Markdown sa mga komento, na maaaring magin mahirap na mahanap at para ma-embed ang mga URL, ang GitHub ay nagpapahintulot sa iyo na i-drag at ihulog ang mga larawan sa mga text area upang ma-embed ang mga ito.
 
```

Comment:
"ito ay hindi kapani-paniwala kapaki-pakinabang" I thinki it it better to say it like this  "ito ay lubhang kapaki-pakinabang"